src/group.c

```diff
@@ -75,13 +75,17 @@ void group_add_member(struct group* group, struct bar_item* item) {
 }
 
 uint32_t group_get_length(struct group* group, struct bar* bar) {
-  int len = group->last_window->origin.x
-            + group->last_window->frame.size.width
-            + group->last_item->background.padding_right
-            + group->first_item->background.padding_left
-            - group->first_window->origin.x;
+    if (group->num_members == 0) return 0;
 
-  return max(len, 0);
+    struct bar_item* bracket = group->members[0];
+
+    int left_edge  = group->first_window->origin.x - bracket->background.padding_left;
+    int right_edge = group->last_window->origin.x
+                   + group->last_window->frame.size.width
+                   + bracket->background.padding_right;
```

Comment:
I am not sure that this actually does what would be expected behavior of the background padding. The background padding is a padding applied *around* the background, i.e. in the bracket case this would be an additional padding *around* the bracket. From what I see the implementation here simply applies an additional padding *inside* the bracket. 

However, there is a problem with having "actual" bracket paddings (see https://github.com/FelixKratz/SketchyBar/issues/351#issuecomment-1518829419)